### DISCUSSION TITLE
How i can load RigidBody in order

### DISCUSSION THREAD

**Discussion by huyprowow** [pmndrs/react-three-rapier#522] [General]
Do you know how I can load RigidBody in order? Currently, when I load a 3D map model and character (gltf), the rigid body of the map is loaded later => the body's rigid body gets stuck on the model, then they collide and my model is thrown away :v (The character is pulled down by gravity in Physic)

**Comment by isaac-mason** [comment]
I work around this by pausing physics while assets are loading. This isn't the most robust method, but works for a simple scene.

See:
- https://github.com/isaac-mason/sketches/blob/585edd6db53e83823e38dc0cbcb194785aeb2fbb/src/sketches/rapier/kinematic-character-controller/index.sketch.tsx#L318
- https://github.com/isaac-mason/sketches/blob/585edd6db53e83823e38dc0cbcb194785aeb2fbb/src/common/hooks/use-loading-assets.ts#L4

**Reply by GTMtremolo** [reply]
Thank you Isaac, 
You save my day
